packit/cli/utils.py

```diff
@@ -109,13 +109,42 @@ def get_packit_api(
         local_project.working_dir, try_local_dir_last=True
     )
 
+    remote_urls = local_project.git_repo.remote().urls
+
+    lp_upstream = None
+    lp_downstream = None
+    for url in remote_urls:
+        if package_config.upstream_project_url:
+            if package_config.upstream_project_url in url:
+                lp_upstream = local_project
+                logger.info("Input directory is an upstream repository.")
+            else:
+                lp_downstream = local_project
+                logger.info(
+                    "Input directory is a downstream repository "
+                    "(upstream url does not match git remote)."
+                )
```

Comment:
I've added `upstream_project_url: https://github.com/packit-service/packit` in `.packit.yaml` and `packit status` tells me that it's downstream - probably because
```
$ git remote -v
upstream	git@github.com:packit-service/packit.gi
```

Reply:
Good point, I need to make it more general.

(note to myself: parse the URL as we do in LocalProject/ogr and compare the server.)

Reply:
Can you try that now, please?

Reply:
Works OK now in the upstream repo, but I'm seeing issues when running in dist-git repo.
I'll create another thread.